iosApp/iosApp/Pages/Search/SearchResultView.swift

```diff
@@ -57,14 +69,34 @@ struct SearchView: View {
         }
     }
 
+    func handleStopTap(stopId: String) {
+        guard let stop = globalResponse?.stops[stopId] else { return }
+        nearbyVM.navigationStack.removeAll()
```

Comment:
Yeah this isn't actually necessary, I added it before we'd decided how to handle hiding the search bar, because I wasn't sure if we'd be able to select a second stop while already on a stop or trip page, and we'd want to clear the stack then (this is the same thing that we're doing when you tap a stop on the map). But yeah, I'll get rid of it for now. It doesn't actually make any difference for the search result navigation, currently the search isn't handled by the navigation, it's a separate state field that persists through navigation, so clearing the nav stack just brings you back to nearby transit with your last search term still populated.